<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FieldPatterns</class>
 <widget class="QMainWindow" name="FieldPatterns">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Field Patterns</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="1,10,0">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tab_selector">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="Checkerboard">
       <attribute name="title">
        <string>Checkerboard</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="GridWidget" name="graph" native="true">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="FieldPattern">
       <attribute name="title">
        <string>FieldPattern</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="FieldPatternWidget" name="fieldpatterngraph" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="rows_label">
      <property name="text">
       <string>Rows</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="graph_label">
      <property name="text">
       <string>Graph</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="rows_setter">
      <property name="minimum">
       <number>2</number>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="column_label">
      <property name="text">
       <string>Columns</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDoubleSpinBox" name="angle1_setter">
      <property name="value">
       <double>45.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QDoubleSpinBox" name="angle2_setter">
      <property name="value">
       <double>45.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="startPoint_setter">
      <property name="value">
       <double>0.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="3">
     <widget class="QPushButton" name="refresh_button">
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="startPoint_label">
      <property name="text">
       <string>Start Point</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="angle1_label">
      <property name="text">
       <string>Angle 1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="column_setter">
      <property name="minimum">
       <number>2</number>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="angle2_label">
      <property name="text">
       <string>Angle 2</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuapp_name">
    <property name="title">
     <string>Wave Propogation Visualization</string>
    </property>
   </widget>
   <addaction name="menuapp_name"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GridWidget</class>
   <extends>QWidget</extends>
   <header>gridwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FieldPatternWidget</class>
   <extends>QWidget</extends>
   <header>fieldpatternwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
